engine/language-server/src/bench/scala/org/enso/languageserver/text/RopeBench.scala

```diff
@@ -1,6 +1,5 @@
 package org.enso.languageserver.text
-import org.enso.languageserver.data.buffer.Rope
-import org.scalacheck.Gen.Parameters
+import org.enso.text.buffer.Rope
```

Comment:
:O Did we not run these on CI? Did we start running these on CI?

Reply:
We didn't, let's start running these on CI! Or at least compile them.

Reply:
Added a compilation check